admin/app/components/solidus_admin/products/index/component.html.erb

```diff
@@ -21,6 +21,7 @@
     search_page_url: search_page_url,
     previous_page_url: previous_page_url,
     next_page_url: next_page_url,
+    turbo_frame_id: 'products-list',
```

Comment:
I have some doubts about passing the `turbo_frame_id` explicitly from outside. It seems something related to the internal implementation that shouldn't be visible from the outside. What if we call it just `id` on the interface and use it internally for all the turbo-related stuff?

Reply:
Related, Rails has these [`dom_id`](https://api.rubyonrails.org/classes/ActionView/RecordIdentifier.html#method-i-dom_id) and [`dom_class`](https://api.rubyonrails.org/classes/ActionView/RecordIdentifier.html#method-i-dom_class) helpers which we could use to generate an HTML ids for resources. See also https://boringrails.com/articles/rails-dom-id-the-most-underrated-helper/ .

Reply:
🙏 I was looking for a way to remove that id. I initialized the id inside the initializer of the `ui/table`. Let me know what you think.